src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/ChainedExpression.cs

```diff
@@ -52,6 +55,8 @@ public class ChainedExpression : Expression
         /// </summary>
         public string TargetResourceType { get; }
 
+        public bool Reversed { get; set; }
```

Comment:
Should this flag be exposed in the ToString below?

Reply:
It should... I had that change sitting on my local version and had apparently forgotten to push. Updated now.